### PR TITLE
Fix Firefox toImage failures

### PR DESCRIPTION
**Pull request by etpinard**
@alexcjohnson @bpostlethwaite @mdtusz @cldougl 

One nasty bug.

### CODE REVIEW

File: src/snapshot/tosvg.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ module.exports = function toSVG(gd, format) {
             // Is this an IE thing? Any other attributes or style elements that can have quotes in them?
             // TODO: this looks like a noop right now - what happened to it?
             var ff = txt.style('font-family');
-            if(ff && ff.indexOf('"') !== -1) txt.style('font-family', ff.replace(/"/g, '"'));
+            if(ff && ff.indexOf('"') !== -1) txt.style('font-family', ff.replace(/"/g, '\\\''));
```
Comment (etpinard) [inline src/snapshot/tosvg.js]: @alexcjohnson could you comment of the comment above this line? 

This patch makes toImage work in FF 42.

**Comment by etpinard** [comment]
This codepen was pretty helpful in figuring out this bug: http://codepen.io/etpinard/pen/bEdQWK

File: src/snapshot/tosvg.js (hunk 1 above)
Comment (bpostlethwaite) [inline src/snapshot/tosvg.js]: lets put a lot of comments here . I would link the codepen example and bug issue and even this PR # in the code. In one year when something else goes wrong here we will be glad of it :)

Comment (alexcjohnson) [inline src/snapshot/tosvg.js]: I'm thinking this was needed for when we passed svg to the back end for image gen... Then we turned it to a noop when we started doing this same fix on the back end... Then it became moot when we started passing json.

**Comment by bpostlethwaite** [comment]
includes a codepen :)

double :dancers: !

**Comment by bpostlethwaite** [comment]
FYI @cpsievert this **might** fix the Rstudio png bug

**Pull request merged**